deps/v8/include/v8.h

```diff
@@ -1701,7 +1701,18 @@ class V8_EXPORT ScriptCompiler {
       Local<String> arguments[], size_t context_extension_count,
       Local<Object> context_extensions[],
       CompileOptions options = kNoCompileOptions,
-      NoCacheReason no_cache_reason = kNoCacheNoReason);
+      NoCacheReason no_cache_reason = kNoCacheNoReason) {
+    return CompileFunctionInContext(
+        context, source, arguments_count, arguments, context_extension_count,
+        context_extensions, options, no_cache_reason, nullptr);
+  };
```

Comment:
This needs to be defined in a `.cc` file to keep ABI compatibility (because consumers are currently expecting the node binary and not the headers to provide the function).

Reply:
Ack

Reply:
@addaleax is it better now?

Reply:
Yes, this should work just fine :)